### PR TITLE
Add TableDnD@0.9.1 w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10712
@cdnjs/intern3 Please help me review this pull request, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/isocra/TableDnD
 * License and its reference: [MIT](https://github.com/isocra/TableDnD/blob/master/MIT-LICENSE.txt)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 35
   - Count of stars: 302
   - Count of forks: 350

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@extend1994 don't forget to update your branch base before sending it out.

**Comment by PeterDaveHello** [comment]
@extend1994 as you can see, GitHub will also remind you here:
https://github.com/extend1994/cdnjs/tree/TableDnD

**Comment by extend1994** [comment]
@PeterDaveHello I rebased it though master branch is not latest. But the gap between them is longer than 2 hours. I thought it's okay for the test? But for recent PR, the similar fail reason kept appearing (Didn't detect sparse-checkout config, should be created from previous stage!). I am sure I have enabled sparseCheckout. After I update the master and rebase, or redo the commits, the test would pass. I am not certain the reason why this happened.

**Comment by PeterDaveHello** [comment]
@extend1994 you should keep it the latest before you send it, that's not a 2 hours or 2 days problem, for the build fails problem, you should let me know which one it is so I can take care of it(sounds like a bug).

**Comment by extend1994** [comment]
@PeterDaveHello Okay I will keep it latest! If next time the build problem still happens, I will report it to you.

**Comment by PeterDaveHello** [comment]
Looks like it's not a bug but a connection problem, will improve on this part soon.

**Comment by PeterDaveHello** [comment]
@extend1994 the problem should be fixed, looks like it just happened 4 hours ago, thanks for report.

BTW, the sparseCheckout means the CI environment, it has nothing to do with your own environment, and as you can see in the log, the problem starts from wget.

File: ajax/libs/TableDnD/bower.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (hare1039) [inline ajax/libs/TableDnD/bower.json]: I don't think we need `bower.json`

**Comment by extend1994** [comment]
@PeterDaveHello OK, thanks for explanation!
@hare1039 I accidentally include the file. Now I removed it, please help me review it, thank you.

File: ajax/libs/TableDnD/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "TableDnD",
+  "filename": "jquery.tablednd.min.js",
+  "version": "0.9.1",
+  "description": "jQuery plug-in to drag and drop rows in HTML tables",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/isocra/TableDnD.git"
+  },
```
Comment (hare1039) [inline ajax/libs/TableDnD/package.json]: How about `homepage` `https://isocra.github.io/TableDnD/`

**Comment by extend1994** [comment]
@hare1039 nice suggestion, thanks. Now please help me review it again.

**Pull request merged**